sdk/aqueduct/utils.py

```diff
@@ -184,6 +186,8 @@ def _package_files_and_requirements(
         file_dependencies:
             Paths of file dependencies the function uses. Note that the paths are relative to the
             file the function is defined in.
+        reqs:
+            An absolute path of file that specify the requirements of the operator.
```

Comment:
but it can also be relative right? Like I can do `../sibling_dir/requirements.txt`